src/docfx/lib/markdown/validation/DocsValidationExtension.cs

```diff
@@ -153,6 +154,21 @@ internal static class DocsValidationExtension
             }
         }
 
+        private static int GetFencedCodeBlockNetLineNumber(FencedCodeBlock fencedCodeBlock)
+        {
+            int netLineNum = 0;
+
+            for (int i = 0; i < fencedCodeBlock.Lines.Count; i++)
+            {
+                if (!string.IsNullOrWhiteSpace(fencedCodeBlock.Lines.Lines[i].ToString()))
```

Comment:
can we avoid ToString to determine whether it's empty？

Reply:
I just confirm it, maybe not.

Reply:
`StringSlice.Trim` method should do the trick without calling `ToString`

Reply:
OK. Got it.